specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2024-06-30-preview/cloudhsm.json

```diff
@@ -0,0 +1,1510 @@
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/cloudHsmClusterName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "backupRequestProperties",
+            "in": "body",
+            "description": "Backup Operation Required properties",
+            "schema": {
+              "$ref": "#/definitions/BackupRequestProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Cloud HSM Cluster Backup operation API finished synchronously with Success",
+            "schema": {
+              "$ref": "#/definitions/BackupResult"
+            },
+            "headers": {
+              "x-ms-request-id": {
+                "description": "A unique ID for the current operation, service generated. All the resource providers must return this value in the response headers to facilitate debugging.",
+                "type": "string"
+              }
+            }
+          },
+          "202": {
+            "description": "Queued validating pre backup operation",
+            "headers": {
+              "Retry-After": {
+                "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
+                "type": "integer",
+                "format": "int32"
+              },
+              "Azure-AsyncOperation": {
+                "description": "The URI to poll for completion status",
+                "type": "string",
+                "format": "uri"
+              },
+              "Location": {
+                "description": "The URI to poll for completion status",
+                "type": "string",
+                "format": "uri"
+              },
+              "x-ms-request-id": {
+                "description": "A unique ID for the current operation, service generated. All the resource providers must return this value in the response headers to facilitate debugging.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "The error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v6/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CloudHsmCluster_ValidateBackup_Validation_MaximumSet_Gen": {
+            "$ref": "./examples/CloudHsmCluster_CreateOrValidate_Backup_MaximumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HardwareSecurityModules/cloudHsmClusters/{cloudHsmClusterName}/backup": {
```

Comment:
>backup [](http://example.com/codeflow?start=155&length=6)

Please keep in mind that by choosing to model these as POSTS , you will not be able to trigger these in ARM templates or invoke these as part of ARM policy affects . Is this acceptable for your scenarios ?

Reply:
Hi @raosuhas, what is  ARM policy affect?